src/angular-pickadate.js

```diff
@@ -249,6 +249,7 @@
           defaultDate: '=',
           minDate: '=',
           maxDate: '=',
+          dayNames: '=?',
```

Comment:
Hi!
The problem there that we can't pass array via '@' scope bindings, in that case we have to pass formatted string, it's more complicated for the usage, I wanted to keep code simple.
Anyway one watcher won't give meanful impact on the performance.